cmd/main.go

```diff
@@ -22,6 +22,7 @@ func main() {
 
 	commands.Version = version
 	commands.Copyright = copyRight
+	utils.SetUserAgent("cpackget/" + version)
```

Comment:
would this be too long?
CMSIS-Toolbox cpackget

Reply:
It's not too long, browsers set User Agents significantly longer.
From this article from [learn.microsoft.com](https://learn.microsoft.com/en-us/previous-versions/windows/internet-explorer/ie-it-pro/internet-explorer-7/bb496341(v=technet.10)?redirectedfrom=MSDN):
> The recommended maximum size for the user agent string is 200 characters.

Are you also suggesting to strip out the version? I would advice to keep it since it's common practice and brings useful information: `CMSIS-Toolbox cpackget/<version>`

Reply:
No, please keep the version. Can we ensure there is always a major.minor.patch version even for none-release versions.

Reply:
The version should always be present even in [nightly](https://github.com/Open-CMSIS-Pack/cmsis-toolbox/blob/main/.github/workflows/nightly.yml#L105) builds (verified in [nightly #200](https://github.com/Open-CMSIS-Pack/cmsis-toolbox/actions/runs/11377216056))

Reply:
```suggestion
	utils.SetUserAgent("CMSIS-Toolbox cpackget/" + version)
```